javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/AgentInstaller.java

```diff
@@ -175,6 +186,13 @@ private static void installBytebuddyAgent(
     }
   }
 
+  private static void copyNecessaryConfigToSystemProperties(InstrumentationConfig config) {
+    String value = config.getString("otel.instrumentation.experimental.span-suppression-strategy");
+    if (value != null) {
+      System.setProperty("otel.instrumentation.experimental.span-suppression-strategy", value);
+    }
```

Comment:
If there are distros setting the default value of this, we need to copy the value over to system properties so that `instrumentation-api` code can get the correct value.